Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -30,17 +30,22 @@ class ZoomPageBar: UIView {
         static let upperZoomLimit: CGFloat = 2.0
         static let zoomInButtonInsets = UIEdgeInsets(top: 6, left: 0, bottom: 6, right: 12)
         static let zoomOutButtonInsets = UIEdgeInsets(top: 6, left: 12, bottom: 6, right: 0)
+        static let startCGColor = UIColor(red: 58/255, green: 57/255, blue: 68/255, alpha: 0.0).cgColor
```

Comment:
Will the colors be the same in light and dark mode?

Reply:
Also should this be something provided by the theme?

Reply:
Hey guys, I have provided the colours for the gradient under dark mode [here](https://www.figma.com/file/dIdFRmA0yACjkam2mb8eGq?node-id=2920:25052#412338376).